### ISSUE TITLE
[BUG?] Restart ActionScript/MXML Language Server Command Not Working At First Run

### ISSUE THREAD

**Issue by angelhdz** [BowlerHatLLC/vscode-as3mxml#428]
### Easy to describe

I have to run the **ActionScript: Run ActionScript/MXML server** command twice in order to make it run.

Extension version: v0.23.2
VSCode version: 1.40.1
AIR version (if needed?): 33.0.2.246

**Comment by joshtynjala** [comment]
I use this command frequently, and I've not encountered the same issue. This bug seems likely to be caused by something specific to your project that doesn't exist in any of mine.

**Comment by joshtynjala** [comment]
@angelhdz Were you able to figure out what was causing the issue?

**Comment by angelhdz** [comment]
Probably VSCode. I'm used to all kind of issues with it. They blame the extensions I use, and the developers  of the extensions I use (Dart/Flutter, Python, etc.) blame VSCode... so I'm now trying IntelliJ...

The issue was, I ran **ActionScript: Run ActionScript/MXML server**, and nothing happened, the errors in JS files like launch.json continued and there was no message that the server was restarting. Then I had to run it a second time, then the server restarted, and the errors were gone.

I need to check if the issue is now fixed in the latest VSCode version. Will check now. Thanks for catching up to this issue! 

Cheers
